apps/web/modules/d/[link]/d-type-view.tsx

```diff
@@ -33,5 +33,3 @@ export default function Type({
     </main>
   );
 }
-
-Type.isBookingPage = true;
```

Comment:
not needed as it will be handled by `useIsBookingPage()` hook used in `app-providers-app-dir.tsx`